フロントエンドのデプロイエラー対応

```
Run npm run deploy

> effective-yomimono@0.1.0 deploy
> npm run deploy:setup && opennextjs-cloudflare build && opennextjs-cloudflare deploy


> effective-yomimono@0.1.0 deploy:setup
> bash ./setup-deploy.sh

Creating KV namespace for Next.js incremental cache...
Found existing KV namespace with ID: 6a21d121b60f43[8](https://github.com/ryosuke-horie/effective-yomimono/actions/runs/15481168479/job/43587150158#step:7:9)5[9](https://github.com/ryosuke-horie/effective-yomimono/actions/runs/15481168479/job/43587150158#step:7:10)932929bd06a183f
KV namespace ID: 6a21d121b60f43859932929bd06a183f
Updated wrangler.jsonc with KV namespace ID
node:internal/modules/cjs/loader:1212
  const err = new Error(message);
              ^

Error: Cannot find module '@ast-grep/napi-linux-x64-gnu'
Require stack:
- /home/runner-user/yomimono-runnner/_work/effective-yomimono/effective-yomimono/frontend/node_modules/@ast-grep/napi/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1212:15)
    at Module._load (node:internal/modules/cjs/loader:[10](https://github.com/ryosuke-horie/effective-yomimono/actions/runs/15481168479/job/43587150158#step:7:11)43:27)
    at Module.require (node:internal/modules/cjs/loader:1298:19)
    at require (node:internal/modules/helpers:182:18)
    at Object.<anonymous> (/home/runner-user/yomimono-runnner/_work/effective-yomimono/effective-yomimono/frontend/node_modules/@ast-grep/napi/index.js:190:31)
    at Module._compile (node:internal/modules/cjs/loader:1529:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1613:10)
    at Module.load (node:internal/modules/cjs/loader:[12](https://github.com/ryosuke-horie/effective-yomimono/actions/runs/15481168479/job/43587150158#step:7:13)75:32)
    at Module._load (node:internal/modules/cjs/loader:1096:12)
    at cjsLoader (node:internal/modules/esm/translators:298:[15](https://github.com/ryosuke-horie/effective-yomimono/actions/runs/15481168479/job/43587150158#step:7:16)) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner-user/yomimono-runnner/_work/effective-yomimono/effective-yomimono/frontend/node_modules/@ast-grep/napi/index.js'
  ]
}

Node.js v20.[19](https://github.com/ryosuke-horie/effective-yomimono/actions/runs/15481168479/job/43587150158#step:7:20).2
Error: Process completed with exit code 1.
```

上記はGitHub Actionsのログ。

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -23,7 +23,7 @@ jobs:
         working-directory: ./api
 
       - name: Frontend Install dependencies
-        run: npm ci
+        run: npm ci --include=optional
         working-directory: ./frontend
 
       - name: Deploy API

